### PR TITLE
feat: add issue templates

### PR DESCRIPTION
**Pull request by barbmarcio**
This PR adds Issue Templates.

You can see what it would look like [here](https://github.com/barbmarcio/rando/issues/new/choose)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+    attributes:
+      label: Expected behavior
+      description: What is the expected behavior? Why is that the expected behavior?
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
+    attributes:
+      label: Screenshots
+      description: Provide screenshots if applicable.
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:33]: this is an api, i don't think screenshots are helpful

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 2]
```diff
@@ -0,0 +1,57 @@
+name: 🪲 Bug report
+description: Create a report to help us improve
+title: "[BUG] "
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:3]: no need to do this if the label is set

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thank you for reporting an issue.
+
+        Please fill in as much of the following form as you're able.
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:10]:
```suggestion
        Please complete the below form to help us look into the issue.
        
        Note: if you are looking for help, create a [discussion](https://github.com/Topsort/topsort.js/discussions) instead.
```

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+      label: SDK Version
+      description: Provide the SDK version (e.g., 0.2.0).
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        **Environment (please complete the following information):**
+        - OS: [e.g. iOS, Android, Windows]
+        - Browser [e.g. Chrome, Safari]
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:46]: or runtime

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+        - Version [e.g. 22]
+  - type: input
+    id: environment
+    attributes:
+      label: Environment
+      description: Provide details about the environment (OS, Browser, Version).
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:52]: i think we can merge this with the above field

File: .github/ISSUE_TEMPLATE/2-feature-request.yml [hunk 6]
```diff
@@ -0,0 +1,43 @@
+name: 💡 Feature request
+description: Suggest an idea for this project
+title: "[FEATURE] "
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/2-feature-request.yml:3]: lets lean on a issue label instead

**Review by jbergstroem — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/1-bug-report.yml [hunk 7]
```diff
@@ -0,0 +1,53 @@
...
+      label: SDK Version
+      description: Provide the SDK version (e.g., 0.2.0).
+    validations:
+      required: true
+  - type: markdown
+    attributes:
+      value: |
+        **Environment (please complete the following information):**
+        - OS: [e.g. iOS, Android, Windows]
+        - Browser [e.g. Chrome, Safari] (or runtime) [e.g. Node.js, .NET, Java]
```
Comment (jbergstroem) [inline .github/ISSUE_TEMPLATE/1-bug-report.yml:43]:
```suggestion
        - Browser [e.g. Chrome, Safari] (or runtime) [e.g. Node.js, Deno, Bun]
```

**Comment by PabloReszczynski** [comment]
Some example issue templates that I like:
- Bun's: https://github.com/oven-sh/bun/tree/main/.github/ISSUE_TEMPLATE
- Neovim: https://github.com/neovim/neovim/tree/master/.github/ISSUE_TEMPLATE

**Review by jbergstroem — APPROVED** [review]
We can always improve from here, LGTM.

**Review by PabloReszczynski — APPROVED** [review]

**Pull request merged**